script/openqa-bootstrap-container

```diff
@@ -15,6 +12,14 @@ set -x
 CONTAINER_NAME="openqa1"
 CONTAINER_PATH="/var/lib/machines/${CONTAINER_NAME}"
 
+systemd_run_params="${systemd_run_params:-"-qM $CONTAINER_NAME"}"
+if systemd-run --help | grep '\-P'; then
+    systemd_run_params="$systemd_run_params -P"
+else
+    echo "Your version of systemd-run does not support the '-P' parameter,
+    piped output from the container will not be available here"
```

Comment:
This way of wrapping the string will lead to surplus white-spaces being printed.

Reply:
Well, I wanted the second line to be indented in the output same as is seen in the source, i.e. indented by 4 spaces. Do you have other preferences?

Reply:
Yes, it looks weird when multiple lines of one sentence have different indentation levels.

Reply:
ok, fine. I have deleted the indention on the second line